src/compo/viirs_l1bnc2ioda.py

```diff
@@ -158,6 +166,13 @@ def _read(self):
                 orbit_ad[:] = 1
             self.varAttrs['satelliteAscendingFlag', metaDataName]['description'] = '0=descending, 1=ascending'
 
+            # NASA VIIRS stored reflectance need to divide by
+            # cosine of solar zenith angle to get true reflectance
+            if self.divide_cos_solarza:
```

Comment:
Got it. 
I tried to run the calculation based on an array from 60 to 94 degrees and it doesn't trigger the floating point exception.
How about masking those points where solar_za is over 85?

Reply:
You could easily just say: `if self.divide_cos_solarza and solar_zenith_angle != 90.0:`
I believe that is easiest and safest.

Reply:
Okay, let the UFO filter screen out those points.
However, it does not trigger the error of division by zero because the value inside np.cos will not exactly equal 0.5pi.
6.123233995736766e-17 is the cosine value when solar_za is 90.
So, it should be fine.

Reply:
Shih-Wei.  Unfortunately, you cannot count on the value being E-17 or similar depending on architecture and compilers, etc.  I wouldn't trust what you found.  I urge the use of additional conditional to be safe.

Reply:
That is true if the compiler automatically zeros the small value, but I doubt it.
My reason (post below) is np.pi will never be exactly the mathematical pi unless human beings find all the digits of pi and the computer has the precision to store it.
But I can add a mask array rather than `solar_za != 90` because it does not work with an array.